subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/feedback/SubscriptionFeedbackCategoryFragment.kt

```diff
@@ -51,6 +62,18 @@ class SubscriptionFeedbackCategoryFragment : SubscriptionFeedbackFragment(R.layo
         binding.categoryPir.setOnClickListener {
             listener.onUserClickedCategory(PIR)
         }
+
+        lifecycleScope.launch {
+            val pirAvailable = isPirCategoryAvailable()
+            withStarted {
+                binding.categoryPir.isVisible = pirAvailable
+            }
+        }
+    }
+
+    private suspend fun isPirCategoryAvailable(): Boolean {
+        val subscription = authRepository.getSubscription() ?: return false
+        return subscription.productId in listOf(MONTHLY_PLAN_US, YEARLY_PLAN_US)
```

Comment:
Is this because PIR will only be available as a feature in US?

Reply:
Yes, exactly.